invokeai/app/invocations/compel.py

```diff
@@ -205,6 +206,7 @@ def _lora_loader() -> Iterator[Tuple[ModelPatchRaw, float]]:
                 returned_embeddings_type=ReturnedEmbeddingsType.PENULTIMATE_HIDDEN_STATES_NON_NORMALIZED,  # TODO: clip skip
                 requires_pooled=get_pooled,
                 device=TorchDevice.choose_torch_device(),
+                split_long_text_mode=SplitLongTextMode.SENTENCES,
```

Comment:
This is the default behaviour in v2.1.1, but it doesn't hurt to manually specify it.

https://github.com/damian0815/compel/blob/2.1.1/src/compel/compel.py#L34